### ISSUE TITLE
Edit Table Properties button on the toolbar is disabled

### ISSUE THREAD

**Issue by JohnLZeman** [giuspen/cherrytree#1240]
With 0.99.14 and now 0.99.15 when I add the Edit Table Properties button to the toolbar, it remains grayed out and disabled when my cursor is within the table.

However the right click context menu option to edit the table properties works as expected.

**Comment by txe** [comment]
@giuspen , popup menu actions are not supposed to be run by the toolbar. Should I just remove these actions from the list?

**Comment by giuspen** [comment]
@txe you are right, those actions should not be available to be added to the toolbar
